components/ui/header.tsx

```diff
@@ -52,14 +52,14 @@ export default function Header({ mode = "dark" }: { mode?: string }) {
                   Mensalistas
                 </Link>
               </li>
-              {/* <li>
+              <li>
                 <Link
                   href="/parceiros"
                   className="font-medium text-slate-800 hover:text-slate-200 dark:text-slate-400 dark:hover:text-slate-200 px-3 lg:px-5 py-2 flex items-center transition duration-150 ease-in-out"
                 >
                   Parceiros
                 </Link>
-              </li> */}
```

Comment:
We shouldn't re-enable the parceiros page when it's not ready yet 👀 same for `mobile-menu.tsx`